cmd/libs/go2idl/parser/parse.go

```diff
@@ -495,3 +504,14 @@ func (b *Builder) walkType(u types.Universe, useName *types.Name, in tc.Type) *t
 		return out
 	}
 }
+
+func (b *Builder) addFunc(u types.Universe, useName *types.Name, in *tc.Func) *types.Type {
+	name := tcFuncNameToName(in.String())
+	if useName != nil {
+		name = *useName
+	}
+	out := u.Get(name)
```

Comment:
I just realized this is problematic if a function's name is the same as a type's name. I'll submit a patch tomorrow.